src/com/oltpbenchmark/util/PlacementPolicy.java

```diff
@@ -0,0 +1,41 @@
+package com.oltpbenchmark.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+ * For GeoPartitioned clusters, the configuration specifying how the
+ * tables should be placed is a list of cloud.region.zone.min_number_of_replicas
+ * blocks in addition to the total replication factor.
+ * 
+ * This class encapsulates such a placement policy.
+ */
+ 
+public class PlacementPolicy {
+    int num_replicas;
+    List<PlacementBlock> placement_blocks;
+
+    public PlacementPolicy() {
```

Comment:
redundant constructor, variable can be initialized directly.

```
List<PlacementBlock> placementBlocks = new ArrayList<>();
```

BTW why it has `default` access level. Why not making it `private`?

Reply:
Done